src/wrapper.rs

```diff
@@ -337,10 +352,14 @@ impl FieldKind {
                             "self.{}.take().unwrap_or_else({}::default)",
                             result.name, unwrapped_type,
                         ));
-                        format!(
-                            "<{} as ::protobuf::Message>::default_instance()",
-                            unwrapped_type,
-                        )
+                        if protobuf {
+                            format!(
+                                "<{} as ::protobuf::Message>::default_instance()",
+                                unwrapped_type,
+                            )
+                        } else {
+                            format!("{}::new_()", unwrapped_type)
+                        }
```

Comment:
I think we can always use `new_` here and avoid passing `protobuf` to `methods`

Reply:
… which means we no longer need a valid implementation of `default_instance` (if there's no direct usages)?

That's pretty nice!

Reply:
I think there are some uses via rust-protobuf's implementation of other methods, but the implementation of default_instance is trivial in any case (it just calls `new_`)